extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -15,4 +15,11 @@
  */
 
  amp-sticky-ad {
+ 	position: fixed;
+ 	bottom: 0;
+ 	left: 0 !important;
+ 	z-index: 1;
+ 	width: 100%;
+ 	display: none;
+ 	max-height: 100px;
```

Comment:
If they don't change, we need `!important`. But:

1. I'm not sure if `max-height: 100px` is a good idea really (it might be). But maybe something more sensitive to viewport size? E.g. using `vh` or `vmin`?
2. I'm quite confused how this element is supposed to size itself. Does it only allow a single ad and sizes by this ad? What happens if the ad doesn't fit?